extra-dev/packages/coq-bignums/coq-bignums.8.10+beta1/opam

```diff
@@ -0,0 +1,42 @@
+opam-version: "2.0"
+maintainer: "Laurent.Thery@inria.fr"
+homepage: "https://github.com/coq/bignums"
+dev-repo: "git+https://github.com/coq/bignums.git"
+bug-reports: "https://github.com/coq/bignums/issues"
+authors: [
+  "Laurent Théry"
+  "Benjamin Grégoire"
+  "Arnaud Spiwack"
+  "Evgeny Makarov"
+  "Pierre Letouzey"
+]
+license: "LGPL 2"
+build: [
+  [make "-j%{jobs}%"]
+]
+install: [
+  [make "install"]
+]
+remove: ["rm" "-R" "%{lib}%/coq/user-contrib/Bignums"]
```

Comment:
Please remove this line, since OPAM now tracks installed files for uninstallation.

Reply:
Done; thanks @palmskog. I'll remember this for next time.